tests/tools/nnpackage_run/src/h5formatter.cc

```diff
@@ -143,6 +143,8 @@ void H5Formatter::loadInputs(const std::string &filename, std::vector<Allocation
           else
             throw std::runtime_error("model input type is int8. But h5 data type is different.");
           break;
+        case NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED:
+          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
```

Comment:
```suggestion
          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
          break;
```

Reply:
I've introduced similar code (https://github.com/Samsung/ONE/pull/8883/files) but it doesn't make new warning, fortunately. (analysis tool version update?)